modules/core/src/main/java/org/apache/ignite/internal/util/CollectionUtils.java

```diff
@@ -204,6 +204,68 @@ public boolean contains(Object o) {
         };
     }
 
+    /**
+     * Union lists.
+     *
+     * @param lists Lists.
+     * @param <T> Type of the elements of lists.
+     * @return Immutable union of lists.
+     */
+    @SafeVarargs
+    public static <T> List<T> union(List<T>... lists) {
```

Comment:
This is technically not a union. I think a better name would be `concat`

Reply:
You're right, I'll rename them.